### PR TITLE
Fixing scikit-cosmo

### PR DESCRIPTION
**Pull request by rosecers**
Addressing #168 

----
:books: Documentation preview :books:: https://rascaline--176.org.readthedocs.build/en/176/

### CODE REVIEW

File: docs/src/conf.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def setup(app):
     "equistore": ("https://lab-cosmo.github.io/equistore/latest/", None),
     "matplotlib": ("https://matplotlib.org/stable/", None),
     "numpy": ("https://numpy.org/doc/stable/", None),
-    "skcosmo": ("https://scikit-cosmo.readthedocs.io/en/latest/", None),
+    "skcosmo": ("https://scikit-matter.readthedocs.io/en/latest/", None),
```
Comment (Luthaf) [inline docs/src/conf.py:140]:
```suggestion
    "skmatter": ("https://scikit-matter.readthedocs.io/en/latest/", None),
```

Comment (rosecers) [reply docs/src/conf.py:140]: I knew I had to miss one!

**Comment by Luthaf** [comment]
Looking at https://rascaline--176.org.readthedocs.build/en/176/how-to/property-selection.html, there are still some skcosmo reference in the examples, and it looks like intersphinx is broken/we don't have links from the examples anymore? I'll look into the second point

**Comment by rosecers** [comment]
I think I found all instances, let me know if you find anything else!

**Comment by Luthaf** [comment]
> and it looks like intersphinx is broken/we don't have links from the examples anymore? I'll look into the second point

I opened #177 for this.

**Pull request merged**